### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by PickleIsDaBest** [3kh0/ext-remover#133] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Thank you guys for spending your time working on a workaround for the new update!

**Comment by ghost** [comment]
yw

**Comment by 3kh0** [comment]
@PickleIsDaBest you can say thank you by staring this repo and following me

**Reply by PickleIsDaBest** [reply]
I will, thanks!
